deployments/azure-functions-app-container.yml

```diff
@@ -0,0 +1,57 @@
+# Action Requires
+# 1. Setup the AZURE_CREDENTIALS secrets in your GitHub Repository
+# 2. Setup the REGISTRY_USERNAME secrets in your GitHub Repository
+# 3. Setup the REGISTRY_PASSWORD secrets in your GitHub Repository
+# 4. Replace REGISTRY, NAMESPACE, IMAGE, TAG in the following template with proper values
+# 5. Add this yaml file to your project's .github/workflows/
+# 6. Push your local project to your GitHub Repository
+
+name: Linux_Container_Workflow
+
+on:
+  push:
+    branches:
+    - master
+
```

Comment:
It would be great to have permissions on workflow as well as job level like in here https://github.com/actions/starter-workflows/blob/main/deployments/azure-container-webapp.yml#L37